### PR TITLE
Update information about e anal.in

### PR DESCRIPTION
**Pull request by officialcjunior**
**Detailed description**
Added information about  `e anal.in` as advised in Issue [#17236](https://github.com/radareorg/radare2/pull/17236)

**Test plan**
* Review and make sure that the given information is correct

**Closing issues**

Issue [#17236](https://github.com/radareorg/radare2/pull/17236)

### CODE REVIEW

**Comment by Maijin** [comment]
It's missing lot of options see `e anal.in=??`

File: src/analysis/code_analysis.md [hunk 1]
```diff
@@ -258,8 +258,15 @@ There are a few options for this:
 - `anal.limits` - enables the range limits for analysis operations
 - `anal.from` - starting address of the limit range
 - `anal.to` - the corresponding end of the limit range
-- `anal.in` - specify search boundaries for analysis (`io.maps`, `io.sections.exec`, `dbg.maps` and many
-more - see `e anal.in=?` for the complete list)
+- `anal.in` - specify search boundaries for analysis. You can set it to `io.maps`, `io.sections.exec`, `dbg.maps` and many more. For example:
+	- To analyze a specific memory map with `anal.from` and `anal.to`, set `anal.in = dbg.maps`.
```
Comment (trufae) [inline src/analysis/code_analysis.md:262]: dont use tabs

Comment (officialcjunior) [reply src/analysis/code_analysis.md]: Fixed that.
I hope it's alright, now?

**Comment by Maijin** [comment]
Looks good to me

**Pull request merged**